lib/weather.js

```diff
@@ -1,9 +1,11 @@
 const rp = require('request-promise');
 const prettyjson = require('prettyjson');
 const prettyjsonOptions = {};
+const colorType = require('../constants/colors');
+// Weather API config
+require('dotenv').config()
```

Comment:
A dotenv is a great idea, but could you please add an example with the expected keys? 

e.g.,: `.env.example`:
```bash
WEATHER_API_KEY=
```

Additionally, ensure that the `.env` is ignored by git, and `.env.example` is *not* ignored by git.